modules/upi-removing-hosts-from-a-upi-cluster.adoc

```diff
@@ -43,36 +43,17 @@ node/app1 drained
 
 . Delete the `customDeploy` specification from the `BareMetalHost` CR.
 
-.. Edit the `BareMetalHost` CR for the host by running the following command:
+. Delete the host by running the following command:
 +
 [source,terminal]
 ----
-$ oc edit bmh -n openshift-machine-api <host_name>
+$ oc delete bmh -n openshift-machine-api <bmh_name>
 ----
 
-.. Delete the lines `spec.customDeploy` and `spec.customDeploy.method`:
-+
-[source,yaml]
-----
-...
-  customDeploy:
-    method: install_coreos
-----
-
-.. Verify that the provisioning state of the host changes to `deprovisioning` by running the following command:
-+
-[source,terminal]
-----
-$ oc get bmh -A
-----
-+
-.Example output
-[source,terminal]
-----
-NAMESPACE               NAME          STATE                    CONSUMER   ONLINE   ERROR   AGE
-openshift-machine-api   controller1   externally provisioned              true             58m
-openshift-machine-api   worker1       deprovisioning                      true             57m
-----
+[NOTE]
+====
+Deleted hosts are deprovisioned automatically.
+====
 
 . Delete the node by running the following command:
```

Comment:
Serial number changed to 1, should be 4 or 5

Reply:
Updated as suggested.